server/src/main/java/io/druid/segment/realtime/FireHydrant.java

```diff
@@ -120,7 +120,7 @@ public String toString()
   {
     return "FireHydrant{" +
            "index=" + index +
-           ", queryable=" + adapter +
+           ", queryable=" + adapter.getIdentifier() +
```

Comment:
Because `ReferenceCountingSegment` doesn't override Object.toString(), so FireHydrant's String form looks like `.., queryable=ReferenceCountingSegment@hash, ..` - not useful.